tasks/galera.yaml

```
@@ -0,0 +1,58 @@
+---
+- name: Check if grastate.dat exists
+  stat:
+    path: /var/lib/mysql/grastate.dat
+  register: mariadb_grastate
```

Task is not idempotent but read-only so we should use `changed_when: false`.

```suggestion
  register: mariadb_grastate
  changed_when: false
```